stdlib/REPL/test/replcompletions.jl

```diff
@@ -101,6 +101,9 @@ test_scomplete(s) =  map_completion_text(shell_completions(s,lastindex(s)))
 test_bslashcomplete(s) =  map_completion_text(bslash_completions(s,lastindex(s))[2])
 test_complete_context(s) =  map_completion_text(completions(s,lastindex(s),Main.CompletionFoo))
 
+module M32377 end
```

Comment:
I'd be interested to know if it still works with:
```suggestion
module M32377
    using REPL
end
```
and if not, confirmation that it doesn't represent a real issue.

But yes, this looks like a great solution!